.eslintrc.json

```diff
@@ -1 +1,15 @@
-{ "extends": "amex" }
+{
+  "extends": ["amex"],
+  "overrides": [
+    {
+      "files": ["**/__tests__/**", "**/__mocks__/**", "*.spec.js"],
+      "extends": ["amex/test"]
+    },
+    {
+      "files": ["*.md"],
+      "rules": {
+        "no-dupe-keys": 0
```

Comment:
What is the reason for disabling `no-dupe-keys`?

Reply:
There is an example in the `README.md` in this section here:

https://github.com/americanexpress/lumberjack/blob/master/README.md#how-to-use-lumberjack

That uses the `formatter` key twice. That was the main reason for disabling `no-dupe-keys`.

Reply:
I would remove this rule and break up the example in the readme